web/src/screens/private/Groups/ViewGroups.js

```diff
@@ -3,149 +3,198 @@ import PropTypes from 'prop-types';
+      .map(tag => ({ value: tag.id.toString(), label: tag.name }));
 
     return (
       <div className={classes.root}>
+        <div className={classes.actionPanel}>
+          <Typography variant="title">Groups</Typography>
+          <div>
+            <Button
+              variant="raised"
+              size="small"
+              color="primary"
+              component={Link}
+              to="/groups/edit"
+            >
+              Add New Group
+            </Button>
+          </div>
+        </div>
         <Paper className={classes.paper}>
-          <Grid container spacing={0}>
-            <Grid item xs={12} sm={6}>
-              <Tabs value={tab} onChange={this.handleTabChange}>
-                <Tab label="Groups" />
-                <Tab label="My Groups" />
-              </Tabs>
-            </Grid>
-            <Grid item xs={12} sm={6}>
-              <div>
-                <FormControl className={classes.formControl}>
-                  <InputLabel htmlFor="locationFilter">Select Location</InputLabel>
-                  <Select
-                    value={locationFilter}
-                    onChange={e => this.handleSelectChange(e, 'locationFilter')}
-                    inputProps={{
-                      id: 'locationFilter',
-                    }}
-                  >
-                    <MenuItem value="">
-                      <em>None</em>
-                    </MenuItem>
-                    {allLocations.map(location => (
-                      <MenuItem value={location} key={location}>
-                        {location}
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-                <FormControl className={classes.formControl}>
-                  <InputLabel htmlFor="capabilityFilter">Select Capability</InputLabel>
-                  <Select
-                    value={capabilityFilter}
-                    onChange={e => this.handleSelectChange(e, 'capabilityFilter')}
-                    inputProps={{
-                      id: 'capabilityFilter',
-                    }}
-                  >
-                    <MenuItem value="">
-                      <em>None</em>
-                    </MenuItem>
-                    {allCapabilities.map(capability => (
-                      <MenuItem value={capability} key={capability}>
-                        {capability}
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-                <TextField
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'flex-end',
+            }}
+          >
+            <FormControlLabel
+              control={
+                <Switch
+                  checked={this.state.myGroups}
+                  value="myGroups"
+                  onChange={this.handleSwitchChange('myGroups')}
```

Comment:
i don't think you want to call the function as part of the render function here

Reply:
For hardcoding - the plan is to use config at an org level for what tags are enabled, I can move this at a later date - trying to balance delivering something and keeping it generic at the moment.